src/db/mod.rs

```diff
@@ -69,6 +81,23 @@ impl DB {
         }
     }
 
+    /// List all itrs attached to a log
+    fn itr_list(&mut self, name: String) -> Result<String, Error> {
+        if name == ""{
```

Comment:
My only note is that instead of checking if name equals a blank string, can we make `name` have the type `Option<String>`?  Since we'll eventually have to write actual docs for this I want to make sure the function signatures match the intent.

Reply:
Actually, you know what, I'll merge this and make this update during a different pass. This PR is already great. No need to nitpick on my end.